pkgs/development/compilers/elm/packages/node-composition.nix

```diff
@@ -2,7 +2,7 @@
 
 {pkgs ? import <nixpkgs> {
     inherit system;
-  }, system ? builtins.currentSystem, nodejs ? pkgs."nodejs-12_x"}:
+  }, system ? builtins.currentSystem, nodejs ? pkgs."nodejs-14_x"}:
```

Comment:
because of this all nodejs packages will change. That's ok we just should test they build at least.